test/acceptance/features/common.feature

```diff
@@ -33,3 +33,7 @@ Feature: Common
       And the value "[CTXT:url.rawquery]" must be equal to "a=1&b=test"
       And the value "[CTXT:url.query.a]" must be equal to "1"
       And the value "[CTXT:url.query.b]" must be equal to "test"
+
+  @common
+  Scenario: Store own ip in context
+    Given I store own ip in context "context.ip"
```

Comment:
I'd rename the step to:
I store my local ip in context ""

Because we can improve the steps in the future with something like:  I store my public ip in context ""